### DISCUSSION TITLE
Non Newtab replacement

### DISCUSSION THREAD

**Discussion by Perisc0** [k-ivan/visual-bookmarks-chrome#98] [Ideas]
Firstly, congratulations for this amazing extension it was the exactly kind of bookmarks layout I was searching for.

But I'm using Brave and would be better have an option to disable the new tab feature, because Brave rewards at new tab as well. 

Thanks!

**Comment by k-ivan** [comment]
Hello and thanks for your interest!
I should clarify that this parameter is not an option, that is, it is either present in the manifest or not. 

In theory, there is an undocumented possibility to influence this. That is, the default tab will remain from the extension, but the extension itself will decide what to display inside the tab. Since this is a dirty way, there is no guarantee that this will also work in other chromium-based browsers or will work in future browser updates.

When I have time, I'll try this dirty trick, but it seems to me that it will only work in chrome.

**Comment by k-ivan** [comment]
I will refuse it until I see a normal way to combine these options.
